Classes/AbstractDataHandlerListener.php

```diff
@@ -84,6 +84,39 @@ protected function getSubPageIds($pageId)
     }
 
     /**
+     * Checks if a page update will trigger a recursive update of pages
+     *
+     * This can either be the case if some $changedFields are part of the TriggerConfiguration or
+     * columns have explicit been configured via plugin.tx_solr.index.queue.recursiveUpdateFields
+     *
+     * @param int $pageId
+     * @param array $changedFields
+     * @return bool
+     */
+    protected function isRecursivePageUpdateRequired($pageId, $changedFields)
+    {
+        // First check TriggerConfiguration
+        $isRecursiveUpdateRequiredTriggerConfiguration = $this->isRecursiveUpdateRequired($pageId, $changedFields);
```

Comment:
your variable name suggests it contains a configuration, but apparently it's a boolean. Just name it `$isRecursiveUpdateRequired`.